plugins/swagger-routes.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const util = require('../lib/nodejs/hexo-util');
+
+const getFilter = (hexo) => {
+
+  const swaggerStore = require('../lib/nodejs/swagger-store')({hexo});
+
+  return () => {
+    const routes = swaggerStore.getRoute();
```

Comment:
@bhaskarmelkani let's call this method getRoutes

Reply:
Okay